src/Campaigns/Blocks/CampaignTitle/editor.scss

```diff
@@ -11,3 +11,8 @@
         }
     }
 }
+
+// Hide the default Gutenberg's post title block as we don't use that
+.edit-post-visual-editor__post-title-wrapper {
+    display: none;
+}
```

Comment:
Ideally, this should be in an editor stylesheet, but for now, it works to keep it within the `CampaignTitle` block since it is being loaded all the time.

Reply:
We use our title block only on campaign pages. This makes sense as it is loaded only on campaign pages.

Reply:
Yeah, but the styles are not being applied to the CampaignTitle block but to the page title element instead. That's what I meant.